DigitalLearningSolutions.Web/ViewModels/Common/CurrentFiltersViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Common

```

Comment:
I wonder if we want to put all the search/sort/filter common classes under a new folder (maybe SearchSortFilter) as the Common folder is having quite a lot of classes now.

Reply:
I wonder if it would also be worth putting the ViewComponent models into a folder at some point too